tests/gtest/avifreadimagetest.cc

```diff
@@ -82,18 +82,17 @@ TEST(PngTest, RgbColorTypeWithTrnsAfterPlte) {
 }
 
 // Verify we can read a PNG file with PNG_COLOR_TYPE_RGB and a tRNS chunk
-// before a PLTE chunk. libpng considers the tRNS chunk as invalid and ignores
-// it, so the decoded image should have no alpha.
-// This test is disabled because the behavior seemed to have changed starting
-// with libpng 1.6.47.
+// before a PLTE chunk, with no MSan use-of-uninitialized-value warnings in
+// avifImageRGBToYUV(). libpng 1.6.46 or older considers the tRNS chunk as
+// invalid and ignores it, so the decoded image has no alpha. The behavior
+// changed starting with libpng 1.6.47.
```

Comment:
Maybe described what changed, for example "The behavior changed starting with libpng 1.6.47 (the decoded image has alpha)"

Reply:
Done.